src/main/java/com/myapp/dao/ItemDao.java

```diff
@@ -0,0 +1,23 @@
+package com.myapp.dao;
+
+import com.myapp.model.Item;
+
+import java.util.List;
+
+public interface ItemDao {
+	public static final String COMMA= ", ";
```

Comment:
fields in interfaces are public static final by default
so it is redundant to write these modificators